android/src/main/java/com/swmansion/rnexecutorch/utils/ETError.kt

```diff
@@ -2,28 +2,29 @@ package com.swmansion.rnexecutorch.utils
 
 enum class ETError(val code: Int) {
     InvalidModelPath(0xff),
-    
-    // System errors
+    FileWriteFailed(0x67),
```

Comment:
why this value?

Reply:
It's based on errors defined in typescript, errors that are not from executorch are starting from 100.